schema/attributes_schema.cue

```diff
@@ -0,0 +1,15 @@
+package shopify
+
+// Structure for attributes, including recursive definitions for values
+#attribute: {
+	id:   =~"^gid://shopify/Taxonomy/Attribute/\\d+$"
+	name: string
+	values: [...{
+		// id:   "\(id)/\\d+" // CUE can't handle nested checks with regexes yet
```

Comment:
This works locally with `#AttributeGid` defined as a constant. `\()` is used for [interpolation](https://cuelang.org/docs/tutorials/tour/expressions/interpolation/).

```
package shopify

#AttributeGid: "gid://shopify/Taxonomy/Attribute/\\d+"

// Structure for attributes, including recursive definitions for values
#attribute: {
	id:   =~"^\(#AttributeGid)$"
	name: string
	values: [...{
		id:   =~"^\(#AttributeGid)/\\d+$"
		name: string
	}]
}

version: =~"^\\d+.\\d+.\\d+$"
attributes: [...#attribute]
```